src/node/utils/Settings.js

```diff
@@ -430,6 +430,11 @@ exports.importMaxFileSize = 50 * 1024 * 1024;
  */
 exports.enableAdminUITests = false;
 
+/*
+ * Enable case-insensitive pad names.
+ */
```

Comment:
I don't think these comments are all that useful; to me they just add noise. (We don't generate documentation from them or anything like that.)

```suggestion
```

Reply:
Hmm, all other things have comments and I think it makes it easier to grasp what the settings is doing. I also added an example, that's better? Otherwise those settings are completely undocumented.